administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -22,6 +23,61 @@
 $input = $app->input;
 
 $assoc = JLanguageAssociations::isEnabled();
+
+// print_r($input->post);
+
+$component = $input->post->get('component');
+$language = $input->post->get('ref-language');
+
```

Comment:
I realise the lines below should be on the model, how do I do it so that when I submit the form the query gets executed? Thanks.